### DISCUSSION TITLE
Does MONAI have support for sincos Positional Embedding on 3D ViT?

### DISCUSSION THREAD

**Discussion by NoTody** [Project-MONAI/MONAI#6966] [General]
Hi, when I look at class PatchEmbeddingBlock class at https://github.com/Project-MONAI/MONAI/blob/339292cad6a9e352dbada30aa139dd2bb39bfa82/monai/networks/blocks/patchembedding.py#L30, it see like sincos positional embedding is not used? Does MONAI have it in other class?

**Comment by KumoLiu** [comment]
Hi @NoTody, we didn't add support for "sincos positional embedding". Do you have some references? Perhaps I can create an issue marked as a "feature request". Welcome to your contribution as well!
Thanks!

**Reply by NoTody** [reply]
For example, this github has sincos positional embedding for Masked Autoencoder implementation for medical imaging. Adding 2D/3D positional information is most commonly used for ViT training. If you create an issue, I can try to contribute to it.

https://github.com/cvlab-stonybrook/SelfMedMAE/blob/68d191dfcc1c7d0145db93a6a570362de29e3b30/lib/models/mae3d.py#L15C5-L15C5

**Reply by KumoLiu** [reply]
Hi @NoTody, added an issue. https://github.com/Project-MONAI/MONAI/issues/6973. 
You can refer to this contribution guide https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md.
Feel free to ask if meet any question.
Thanks!
